pandas/tests/groupby/test_function.py

```diff
@@ -940,10 +940,6 @@ def test_frame_describe_multikey(tsframe):
     groupedT = tsframe.groupby({"A": 0, "B": 0, "C": 1, "D": 1}, axis=1)
     result = groupedT.describe()
     expected = tsframe.describe().T
-    expected.index = pd.MultiIndex(
```

Comment:
> One existing test previously had to manipulate the expected index, but that is no longer necessary.

@smithto1 I don't understand the change to the expected here, can you expand on why the new value is correct?

Reply:
With the bugfix in this PR, I think `.describe()` on `axis=1` is now encountering the bug reported in #26805, #22848 #22546 where `group_keys=True` is ignored. 

The behaviour of the new code with `group_keys=True/False` is the same as the old code with `group_keys=False`.

I'll try to take a look at the 'group_keys' bug; if that is fixed I think this test will probably be reverted.

Reply:
I think this may be addressed by #34998 but for the `axis=1` case.